README.md

```diff
@@ -7,3 +7,9 @@ Time to learn the difference between init and init --bare
 So there will be another line after this pushed with -u
 
 So this is a line after -u
+
+## So this is a branch test
+
+1. Created a new branch and gave it a long and weird name.
+2. Now I'm gonna test how this thing works :)
```

Comment:
What does this smiley face mean huh? 
This is a serious  company, with some serious goals set ahead!

Reply:
No